src/features/rewards/walletWrapper/index.tsx

```diff
@@ -244,6 +244,32 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
     )
   }
 
+  getNotificationButton = (type: NotificationType, onClose: any) => {
+    let buttonText = 'OK'
```

Comment:
please add locale for this one. I think we fixed panel locals now so we can re-introduce it

Reply:
done